src/FreeType/ExternalAddress.extension.st

```diff
@@ -0,0 +1,6 @@
+Extension { #name : #ExternalAddress }
+
+{ #category : #'*FreeType' }
```

Comment:
Should be moved to UnifiedFFI package

Reply:
FFIExternalStructureReferenceHandle>>#asExternalAddress is inside the FreeType package (https://github.com/pharo-project/pharo/blob/Pharo8.0/src/FreeType/FFIExternalStructureReferenceHandle.extension.st). Should both move to UnifiedFFI?

Reply:
Could be nice...

Reply:
Hi, @zecke 
can you perform the recategorisation, so I can integrate this?
Thanks!
Esteban

Reply:
Done. I moved both into UnifiedFFI